### PR TITLE
Fix top navbar resize

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Without the "not latest version":
not-latest

With the "latest version":
latest

### CODE REVIEW

**Review by jyn514 — APPROVED** [review]
This is what it looked like previously:

image
image

**Comment by Nemo157** [comment]
Having a fixed-width name element looks weird with short names, could that be `max-width` instead?

As well as latest version, there's a warning message for yanked crates, it should probably take that into account too.

**Comment by Nemo157** [comment]
Did you try using the right-hand dropdowns when they're in their icon+dropdown form? IIRC they go off the right edge of the screen, which is why I made them disable the dropdown when shrunk.

File: templates/header/topbar_begin.html [hunk 1]
```diff
@@ -9,7 +9,8 @@
 <div class="nav-container">
     <div class="container">
         <div class="pure-menu pure-menu-horizontal" role="navigation" aria-label="Main navigation">
-            <form action="/releases/search" method="GET" class="landing-search-form-nav">
+            <form action="/releases/search" method="GET"
+                {% if not is_latest_version %}class="landing-search-form-nav not-latest"{% else %}class="landing-search-form-nav"{% endif %}>
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply templates/header/topbar_begin.html:13]: I'm still sad that this form doesn't work: `{{ 'not-latest' if not is_latest_version }}` :'(

**Comment by GuillaumeGomez** [comment]
> Having a fixed-width name element looks weird with short names, could that be `max-width` instead?

I tried with max-width but honestly, having an element with a fixed size was much more comfortable (for me at least).

File: templates/style/_navbar.scss [hunk 2]
```diff
@@ -25,6 +25,9 @@ div.nav-container {
         }
         &:after {
             font-size: 0.8em;
+            position: absolute;
```
Comment (Nemo157) [inline templates/style/_navbar.scss:28]: This makes the menu feel very cramped to me, and especially there is not enough padding on the right-hand side of the arrow.

Comment (GuillaumeGomez) [reply templates/style/_navbar.scss:28]: Do you have a screenshot by any chance?

Comment (Nemo157) [reply templates/style/_navbar.scss:28]: Current look:

image

On this branch:

image

Comment (GuillaumeGomez) [reply templates/style/_navbar.scss:28]: Yes, that's exactly how I intended to have it. Do you prefer having the arrow right next to the text maybe?

Comment (Nemo157) [reply templates/style/_navbar.scss:28]: (that screenshot also shows off the extra whitespace after short crate names that I dislike).

Comment (Nemo157) [reply templates/style/_navbar.scss:28]: IMO the whitespace from the left-border to the icon/text and from the drop-down-arrow to the right border should be the same to make it feel balanced.

Comment (GuillaumeGomez) [reply templates/style/_navbar.scss:28]: If you have a solution which allows to have a way to have just a max-width and that the items will remove the display of the text by themselves, I'd be glad to hear it (a lot of CSS remains unknown to me).

Comment (Nemo157) [reply templates/style/_navbar.scss:28]: Changing `width: 200px;` to `max-width: 200px;` appears to work to me.

Comment (GuillaumeGomez) [reply templates/style/_navbar.scss:28]: It "works". However, the items on the right won't remove the text at the "right" time when the windows size is changing on smaller crate titles.

Comment (Nemo157) [reply templates/style/_navbar.scss:28]: You mean that there's extra whitespace available they could use when they shrink? Doesn't seem like a problem to me, the exact amount of whitespace there will be influenced by things like the font selected anyway, and not many people drag window sizes around and notice if the responsiveness is _exactly_ right. As long as the media queries target the maximum that could be used it seems fine.

Comment (GuillaumeGomez) [reply templates/style/_navbar.scss:28]: I guess this is good enough. I'll do that then!

**Review by Nemo157 — APPROVED** [review]

**Pull request merged**